src/Workspaces/Core/Portable/PublicAPI.txt

```diff
@@ -787,6 +787,7 @@ Microsoft.CodeAnalysis.Workspace.ScheduleTask<T>(System.Func<T> func, string tas
 Microsoft.CodeAnalysis.Workspace.Services.get -> Microsoft.CodeAnalysis.Host.HostWorkspaceServices
 Microsoft.CodeAnalysis.Workspace.SetCurrentSolution(Microsoft.CodeAnalysis.Solution solution) -> Microsoft.CodeAnalysis.Solution
 Microsoft.CodeAnalysis.Workspace.UnregisterText(Microsoft.CodeAnalysis.Text.SourceTextContainer textContainer) -> void
+Microsoft.CodeAnalysis.Workspace.UpdateReferencesAfterAdd() -> void
```

Comment:
I was considering baking in a call to it directly, but there is not an ideal place for it to go.  It needs the host to make the decision.